app/javascript/generation-users.vue

```diff
@@ -0,0 +1,147 @@
+<template lang="pug">
+.page-body
+  nav.pagination(v-if='totalPages > 1')
+    pager(v-bind='pagerProps')
+  .container
+    .users
+      .row.is-gutter-width-32(v-if='users === null')
+        .empty
+          .fas.fa-spinner.fa-pulse
+          |
+          | ロード中
+      .row.is-gutter-width-32(v-else-if='users.length !== 0')
+        user(
+          v-for='user in users',
+          :key='user.id',
+          :user='user',
+          :currentUser='currentUser'
+        )
+      .row.is-gutter-width-32(v-else)
+        .o-empty-message
+          .o-empty-message__icon
+            i.far.fa-smile
+          p.o-empty-message__text
+            | {{ generationID }}期のユーザー一覧はありません
+  nav.pagination(v-if='totalPages > 1')
+    pager(v-bind='pagerProps')
+</template>
+<script>
+import User from './user.vue'
```

Comment:
ちょっと試してないんですが、下記だと動かなかったりしますかね？

```suggestion
import User from 'user.vue'
```

Reply:
試してみたところ、`./`無しでも問題なく動きました。(`.vue`ファイルだけでなく`.js`ファイルも)
本PRで変更したファイルから不要な`./`を削除してpushしました。

一つ質問なのですが、こういう動作には不要な`./`はpathに含めないほうが良かったりするのでしょうか？
少し調べてみた感じだと、「プロジェクトに依る」という意見が多かったので...。

もし動作に不要な`./`は含めない方が良いのであれば、別issueを立てて対応しようと思います。

Reply:
@naomichi-h 不要なのでつけなくて大丈夫です。
今回の変更した行については`./`つけないようにして、今回触ってない行について別Issueを立てていただければありがたいです〜

Reply:
@komagata 不要な`./`についてのissue立てました！https://github.com/fjordllc/bootcamp/issues/3997
今回変更した行について、`./`をつけないように修正しましたので、確認をよろしくお願いいたします🙏